modules/@apostrophecms/area/ui/apos/components/AposAreaWidget.vue

```diff
@@ -268,7 +268,12 @@ export default {
       };
     },
     widgetLabel() {
-      return window.apos.modules[`${this.widget.type}-widget`].label;
+      const moduleName = `${this.widget.type}-widget`;
+      const module = window.apos.modules[moduleName];
+      if (!module) {
+        console.error(`No ${moduleName} module found for widget type ${this.widget.type}`);
```

Comment:
No need to throw an exception here, the next line will do it, this just makes it more obvious why